### PR TITLE
Fix color toHexa()

### PR DESCRIPTION
**Pull request by jparkrr**
Need to strip out the decimals.

### CODE REVIEW

**Comment by jparkrr** [comment]
@asturur did you see this one? Currently `color.toHexa()` can give things like `FFFFFF19.8`

**Comment by asturur** [comment]
i saw it, but between wekeend and christmas i did not have time to check it

File: src/color.class.js [hunk 1]
```diff
@@ -192,7 +192,7 @@
       var source = this.getSource(), a;
 
       a = source[3] * 255;
-      a = a.toString(16);
```
Comment (asturur) [inline src/color.class.js]: can't we use math.Round on source[3] * 255 instead of using a split string?

**Comment by jparkrr** [comment]
@asturur Great idea on rounding! Updated

Sorry if I rushed you, take your time. Hope you had a good Christmas!

**Pull request merged**